GVFS/GVFS.Platform.POSIX/POSIXPlatform.cs

```diff
@@ -26,15 +22,15 @@ public abstract partial class POSIXPlatform : GVFSPlatform
 
         public override IGitInstallation GitInstallation { get; } = new POSIXGitInstallation();
         public override IPlatformFileSystem FileSystem { get; } = new POSIXFileSystem();
-
+        public override GVFSPlatformConstants Constants { get; } = new GVFSPlatformConstants(executableExtension: string.Empty, installerExtension: ".dmg");

```

Comment:
Initializer belongs in constructor.

Reply:
As discussed, this can be done in a follow-up PR. Created a [tracking issue](https://github.com/Microsoft/VFSForGit/issues/1083).